src/components/LeaseComponents/AnnualMiles.tsx

```diff
@@ -0,0 +1,27 @@
+import {useContext, useEffect, useState} from "react";
+import { AppContext } from "../../ValueProvider";
+
+
+export default function AnnualMilesField() {
+    const {setMileage} = useContext(AppContext)
+    const [miles,setMiles]= useState<number>(12000)

```

Comment:
Instead of state use context values and dispatchers.

Reply:
Changed.